canvas_sdk/effects/protocol_card/constants.py

```diff
@@ -0,0 +1,15 @@
+from typing_extensions import Literal, NotRequired, TypedDict
+
+
+class RecommendationCodingFilter(TypedDict):
+    """A coding object intended for use in a ProtocolCard Recommendation filter."""
+
+    system: Literal["cpt", "cvx", "snomedct", "rxnorm", "loinc", "icd10cm", "fdb", "ndc"]
+    code: str
```

Comment:
this list came from here: https://github.com/canvas-medical/canvas/blob/develop/home-app/api/graphql/protocol/types.py#L816-L825